__tests__/api.test.js

```diff
@@ -149,3 +149,68 @@ describe("5. PATCH /api/reviews/:review_id", () => {
       });
   });
 });
+
+describe("2. GET /api/reviews", () => {
+  test("status:200, responds with every review in DESC order", () => {
+    return request(app)
+      .get(`/api/reviews`)
+      .expect(200)
+      .then(({ body }) => {
+        let review = body.review;
+        expect(review.length).toEqual(13);
+        expect(review).toBeSortedBy("created_at", {
+          descending: true,
+        });
+        review.forEach(review => {
+          expect(review).toEqual(
+            expect.objectContaining({
+              review_id: expect.any(Number),
+              title: expect.any(String),
+              category: expect.any(String),
+              designer: expect.any(String),
+              owner: expect.any(String),
+              review_body: expect.any(String),
+              review_img_url: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              comment_count: expect.any(String),
+            })
+          );
+        });
+      });
+  });
+});
+
+describe("GET api/treasures where filter matches query", () => {
+  it("should return the caregory that matches the input", () => {
+    return request(app)
+      .get("/api/reviews?category=dexterity")
+      .expect(200)
+      .then(response => {
+        expect(response.body.review[0].category).toBe("dexterity");
+      });
+  });
+  it("should return an array of treasures only containg matched query value", () => {
+    return request(app)
+      .get("/api/reviews?category=social deduction")
```

Comment:
195 - be careful of using spaces in addresses - URLs should not contain them.  Usually + or %20 is used instead.  using _ would also be OK